src/pages/Home/Home.page.jsx

```diff
@@ -1,37 +1,18 @@
 import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
 
-import { useAuth } from '../../providers/Auth';
-import './Home.styles.css';
+import Header from '../../components/Header';
+import ListPreviewVideo from '../../components/Content/ListPreviewVideo';
+import Content from '../../components/Content/Content.component';
 
 function HomePage() {
-  const history = useHistory();
   const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
-
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
 
   return (
     <section className="homepage" ref={sectionRef}>
-      <h1>Hello stranger!</h1>
-      {authenticated ? (
-        <>
-          <h2>Good to have you back</h2>
-          <span>
-            <Link to="/" onClick={deAuthenticate}>
-              ← logout
-            </Link>
-            <span className="separator" />
-            <Link to="/secret">show me something cool →</Link>
-          </span>
-        </>
-      ) : (
-        <Link to="/login">let me in →</Link>
-      )}
+      <Header />
+      <Content>
+        <ListPreviewVideo></ListPreviewVideo>
```

Comment:
Este componente puede cerrarse aquí mismo porque no lleva children.
```suggestion
        <ListPreviewVideo/>
```